src/integrationTest/java/uk/gov/hmcts/reform/ucmc/controllers/BaseIntegrationTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.ucmc.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import uk.gov.hmcts.reform.ucmc.Application;
+
+@SpringBootTest(
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+    classes = Application.class
+)
+@AutoConfigureMockMvc
+public class BaseIntegrationTest {
+
+    protected static final String BEARER_TOKEN = "Bearer let me in";
+
+    @Autowired
+    protected ObjectMapper objectMapper;
+
+    @Autowired
+    protected MockMvc mockMvc;
+
+    protected ResultActions doPost(String auth, String content, String urlTemplate, Object... uriVars)
```

Comment:
how about `content` being CallbackRequest, so we don't have to wrap it in `toJson()` in every subclass (it could be done inside this method)?

Reply:
As this is base class, we may want to pass some other objects as well for different integration tests.